### DISCUSSION TITLE
Teamtalk python sdk: how to stream raw audio data to teamtalk, and how to stream media file via python sdk?

### DISCUSSION THREAD

**Discussion by denizsincar29** [BearWare/TeamTalk5#1740] [Q&A]
Hello!
I am new to python sdk and don't know how to use its low level api.
I want to be able to make text to speech announcements in some channels, but don't know how to stream a media file in python teamtalk sdk, or raw audio data if possible.
can you give me some examples on how to do that in python?
I hope that functionality will be presented in the high level api..
thank you.

**Comment by bear101** [comment]
The Python API is quite limited at the moment. I would suggest you C# or Java if you want a high level API.
